install-deps.sh

```diff
@@ -338,6 +338,7 @@ else
         echo "Using dnf to install dependencies"
         case "$ID" in
             fedora)
+                $SUDO dnf install -y ninja-build
```

Comment:
I am not sure if this is the best location in this script to install this package.

Reply:
@rishabh-d-dave neither do i =). just created #39991 to address this issue.